src/uucore/Cargo.toml

```diff
@@ -52,6 +52,7 @@ blake2b_simd = { workspace = true, optional = true }
 blake3 = { workspace = true, optional = true }
 sm3 = { workspace = true, optional = true }
 regex = { workspace = true, optional = true }
+quick-error = { workspace = true, optional = true }
```

Comment:
Do we still want to use `quick-error`? See https://github.com/uutils/coreutils/pull/6453